src/lib/common/dtoa.h

```diff
@@ -0,0 +1,239 @@
+/*
```

Comment:
This file is already part of radidjson (typically, /usr/local/include/rapidjson/internal/dtoa.h). Maybe it could be used from the library instead of including it as Orion library?

Reply:
I have included this file to change how `rapidjson` render numbers (I am working now on creating a PR in `rapidjson`). I think is a bug to use a different syntax when rendering float and integer numbers. Take into account that JSON only uses one type for numbers (numbers are always floating numbers, there is not a different syntax for integer numbers).

Reply:
The rendering problem is exposed when rendering an integer number. E.g. if you use the `Double` method provided by `rapidjson` to render a `4` number it is rendered as: `4.0`. This syntax is required by some languages (like C/C++) to mark it as a floating number, but in JSON it is not required, so it would be better written as `4`, and is what is usually done by other libraries (in fact, `rapidjson` is the only library I know that serialises numbers using this C/C++ pattern)